panther_driver/src/panther_can.py

```diff
@@ -18,327 +18,337 @@ def __setattr__(self, name, value):
         raise AttributeError(f'can\'t reassign constant {name}')
 
 
-@dataclass
 class MotorController:
-    def __init__(self, can_node_id: int, eds_file: str) -> None:
-        self._last_time_callback = {}
+    HEARTBEAT_TIMEOUT_S = 0.3
```

Comment:
```suggestion
    HEARTBEAT_TIMEOUT = 0.3
```
I think it is generally assumed that time is specified in seconds